ibis/backends/flink/__init__.py

```diff
@@ -412,7 +444,15 @@ def create_view(
         Table
             The view that was created.
         """
-        raise NotImplementedError
+        if obj is None:
+            raise exc.IbisError("The obj parameter is required")
+        
+        if overwrite and self.list_views(name):
+            self.drop_view(name=name, catalog=catalog, database=database, force=True)
+            
+        qualified_name = self._fully_qualified_name(name, database, catalog)
+        self._table_env.create_temporary_view(qualified_name, obj)
```

Comment:
@deepyaman FYI, since we are creating views from the session environment, for this reason, among other potential, the remote cluster execution using existing CLI might be challenging. (we'll also have to check if we have other implementation that directly leverages session environment instead of creating direct SQL statements)